lib/configure-map.js

```diff
@@ -135,10 +135,24 @@ module.exports = function (original, length, options) {
 			return conf.delete(arg);
 		};
 	}
+	extGet = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return cache[id];
+	};
+	extHas = function () {
+		var id, args = arguments;
+		if (resolve) args = resolve(args);
+		id = get(args);
+		return conf.has(id);
```

Comment:
Theoretically we can have `'null'` cache id (e.g. with primitive serialization for one argument taking function, the `null` argument will resolve to `'null'` cache id).
And in such case `hasOwnProperty.call(cache, null)` will result with `true`, as second argument is in all cases coerced by engine to string:

```javascript
cache = { 'null': true };
Object.prototype.hasOwnProperty.call(cache, null); // true
```